### ISSUE TITLE
Nuclei importer

### ISSUE THREAD

**Issue by gdemarcsek** [DefectDojo/django-DefectDojo#3898]
**Scanner Name**
Nuclei (JSON-per-line): https://github.com/projectdiscovery/nuclei (e.g.: https://github.com/projectdiscovery/nuclei/blob/31e9c04d5ec05674bad557ede44f22cecd15c0eb/v2/pkg/executer/output_http.go#L25 , https://github.com/projectdiscovery/nuclei/blob/31e9c04d5ec05674bad557ede44f22cecd15c0eb/v2/pkg/executer/output_dns.go#L16)

**Sample File**
Varies based on the template, / workflow, a simple example that doesn't use any extractor (e.g. WAF detection `nuclei -l target_urls.txt -t ./technologies/waf-detect.yaml -json`)

```json
{"name":"WAF Detection","author":"dwisiswant0","severity":"info","matcher_name":"perimx","matched":"https://example.com/","template":"waf-detect","type":"http","host":"https://example.com/"}
{"severity":"info","matcher_name":"ats","matched":"https://example.com/","template":"waf-detect","type":"http","host":"https://example.com/","name":"WAF Detection","author":"dwisiswant0"}
...
```

**Comment by ansidorov** [comment]
I will make a parser

**Comment by damiencarol** [comment]
Fix in `2.0.0`
